velox/exec/WindowPartition.cpp

```diff
@@ -368,6 +370,23 @@ void WindowPartition::updateKRangeFrameBounds(
   // reordered as per inputMapping_.
   RowColumn frameRowColumn = columns_[frameColumn];
   RowColumn orderByRowColumn = data_->columnAt(orderByColumn);
+
+  // Mark the frame invalid if the frame bound is NaN.
+  auto frameType = data_->columnTypes()[mappedFrameColumn];
```

Comment:
@kagamiori : We don't need these loops anymore.

Reply:
Oh, right. Let me remove them.